packages/terra-tabs/src/Tabs.jsx

```diff
@@ -169,35 +178,44 @@ class Tabs extends React.Component {
       React.cloneElement(contentItem, { isLabelHidden: isIconOnly || this.state.isLabelTruncated })
     ));
 
-    const collapsibleTabs = (
-      <CollapsibleTabs
-        activeKey={activeKey || this.state.activeKey}
-        activeIndex={this.getActiveTabIndex()}
-        onChange={this.handleOnChange}
-        onTruncationChange={this.handleTruncationChange}
-        variant={variant}
-      >
-        {clonedPanes}
-      </CollapsibleTabs>
-    );
+    const tabContent = (() => {
+      const collapsibleTabs = (
+        <CollapsibleTabs
+          activeKey={activeKey || this.state.activeKey}
+          activeIndex={this.getActiveTabIndex()}
+          onChange={this.handleOnChange}
+          onTruncationChange={this.handleTruncationChange}
+          variant={variant}
+        >
+          {clonedPanes}
+        </CollapsibleTabs>
+      );
+
+      if ([DependentViewport.PARENT, DependentViewport.WINDOW].includes(responsiveTo)) {
+        const collapsedTabs = (
+          <CollapsedTabs activeKey={activeKey || this.state.activeKey} onTruncationChange={this.handleTruncationChange}>
+            {clonedPanes}
+          </CollapsedTabs>
+        );
+
+        return (
+          <ResponsiveElement
```

Comment:
If tabs are responsive to none, we should not wrap the tabs in a responsive element and just return `CollapsibleTabs`

Reply:
Ah, i see you are doing this already, Disregard.